projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-input/search-ai-input.component.ts

```diff
@@ -127,20 +126,15 @@ export class SearchAiInputComponent implements OnInit, OnDestroy {
     private agentService: AgentService,
     private userPreferencesService: UserPreferencesService,
     private translateService: TranslateService,
-    private modalAiService: ModalAiService,
-    private route: ActivatedRoute
+    private modalAiService: ModalAiService
   ) {}
 
   ngOnInit(): void {
-    if (this.searchTerm) {
-      this.queryControl.setValue(this.searchTerm);
-    } else if (this.route.snapshot?.queryParams?.query?.length > 0) {
-      this.queryControl.setValue(this.route.snapshot.queryParams.query);
-    } else {
-      this.queryControl.setValue(null);
-    }
+    const usedInAiResultsPage = this.useStoredNodes;
```

Comment:
Could you please rename that input useStoredNodes to usedInAiResultsPage? Actually it looks strange that we assign useStoredNodes to usedInAiResultsPage as the relationship is not clear. It will be easier to understand when we rename whole input (and usages in parents - please see it's used also in SearchAiInputContainerComponent)